lib/livebook/live_markdown/import.ex

```diff
@@ -198,7 +208,7 @@ defmodule Livebook.LiveMarkdown.Import do
   defp build_notebook([{:cell, :elixir, source, outputs} | elems], cells, sections, messages) do
     {metadata, elems} = grab_metadata(elems)
     attrs = cell_metadata_to_attrs(:elixir, metadata)
-    outputs = Enum.map(outputs, &{:text, &1})
+    outputs = Enum.map(outputs, fn output -> if is_tuple(output), do: output, else: {:text, output} end )
```

Comment:
You can wrap the text output in `{:text, output}` upfront, so that we don't need this conversion :)

Reply:
Oh, I completely forgot I could do this. Life is hard when you only have one brain cell. lol
It's solved in f4b3858